dev-client/app.config.ts

```diff
@@ -0,0 +1,71 @@
+import {ExpoConfig, ConfigContext} from 'expo/config';
+
+const appVersion = 82;
```

Comment:
suggestion: can we read this from a text or JSON file?

Reply:
what i would like to implement but haven't yet implemented is to read this from an environment variable. then our github workflows can provide it based on commit tags, so rather than creating and merging a commit updating the value in a config file, the codebase itself has no idea what version it is and we can push a tag `v86` to a branch to tell the CI pipeline to build that commit as build number 86. this will also be a necessary step for e.g. building multiple different apps from one codebase